resolvers/npm_cache/lib.rs

```diff
@@ -299,21 +311,23 @@ impl<
   pub fn load_package_info(
     &self,
     name: &str,
-  ) -> Result<Option<NpmPackageInfo>, serde_json::Error> {
+  ) -> Result<Option<SerializedCachedPackageInfo>, serde_json::Error> {
     let file_cache_path = self.get_registry_package_info_file_cache_path(name);
 
-    let file_text = match std::fs::read_to_string(file_cache_path) {
+    let file_bytes = match self.sys.fs_read(&file_cache_path) {
       Ok(file_text) => file_text,
       Err(err) if err.kind() == ErrorKind::NotFound => return Ok(None),
       Err(err) => return Err(serde_json::Error::io(err)),
     };
-    serde_json::from_str(&file_text)
+
+    // todo(dsherret): deserialize in a blocking task
+    serde_json::from_slice(&file_bytes)
```

Comment:
Is there a reason you want to do it in a blocking task? If so, should it be done in this PR?

Reply:
Because we can parallelize it like we do elsewhere. I want to see and measure its affects in a followup pr.